meson.build

```diff
@@ -476,6 +476,26 @@ if elogind.found()
   conf.set('HAVE_LOGIND' , '1')
 endif
 
+if get_option('launchd').disabled()
+  use_launchd = false
```

Comment:
this block is no longer necessary I think.  It will disable automatically when it's not needed AFAICT.